### PR TITLE
Add port for Nucleo-F767ZI

### PR DESCRIPTION
**Pull request by jnugen**
Related: https://github.com/micro-ROS/freertos_apps/pull/44

### CODE REVIEW

File: config/freertos/nucleo_f767zi/package.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>firmware</name>
+  <version>0.0.0</version>
+  <description>Micro-ROS dependecies for FreeRTOS and STM32F767ZI Nucleo Board</description>
+  <maintainer email="pablogarrido@eprosima.com">Eden Desta</maintainer>
```
Comment (pablogs9) [inline config/freertos/nucleo_f767zi/package.xml:7]: I guess that we don't have the same mail.

**Review by pablogs9 — CHANGES_REQUESTED** [review]

**Comment by pablogs9** [comment]
Is possible to add this board to the CI?

**Comment by pablogs9** [comment]
Also add this board to the README.md tables of supported hardware please.

File: config/freertos/nucleo_f767zi/package.xml (hunk 1 above)
Comment (jnugen) [reply config/freertos/nucleo_f767zi/package.xml]: Sorry, I missed that one.

**Review by pablogs9 — APPROVED** [review]

**Pull request merged**